b2share/modules/records/minters.py

```diff
@@ -77,7 +77,7 @@ def b2share_oaiid_minter(rec_pid, data):
     )
     data.setdefault('_oai', {})
     data['_oai'].update({
-        'id': rec_pid.pid_value,
+        'id': oai_pid_value,
```

Comment:
Why are we not using `[oaiid_minter](https://github.com/inveniosoftware/invenio-oaiserver/blob/master/invenio_oaiserver/minters.py#L38)`? The only thing we do in addition here is set the `sets`. We could just call `oaiid_minter` and then update the sets (even if it is not the best place to do it).
You can see how Zenodo uses it [here](https://github.com/zenodo/zenodo/blob/master/zenodo/modules/records/minters.py#L79)  and [here](https://github.com/zenodo/zenodo/blob/d0bfb7edd06141483f1c01bc9048a7810cc7ec60/zenodo/config.py#L880)

Reply:
As you say, it's basically the same code, it's short and it works, so why change it now? We'd also probably need to mess with some configs.

Reply:
Yes we can keep it for now. I was just wondering if there was a specific reason as I couldn't find any customization. We can keep this as an issue for the final code cleaning.